### DISCUSSION TITLE
Google coral dev board

### DISCUSSION THREAD

**Discussion by VX-36998120** [weltenwort/frigate-synology-dsm7#54] [General]
Hello, is there any way to use dev board in synology. I really can't buy Google coral USB.

**Comment by weltenwort** [comment]
Hey @siri10086, I don't think that would work. The dev board is a SOC in itself and not a periphery device. But what you could try is to run frigate on the dev board directly. It runs a Debian derivative (Mendel), which should provide the necessary dependencies.

**Reply by VX-36998120** [reply]
Ok thanks
